### PR TITLE
Disable color if stout is not a tty

### PR DESCRIPTION
**Pull request by leiserfg**

### CODE REVIEW

File: src/client.rs [hunk 1]
```diff
@@ -1,9 +1,12 @@
+extern crate atty;
```
Comment (saghm) [inline src/client.rs]: The crate has been updated to the 2018 edition of Rust, so this line shouldn't be necessary.

**Review by saghm — COMMENTED** [review]
Hi! Thanks so much for the pull request! I apologize for not responding in a timely fashion.

This all looks really good! I have one minor suggestion, but other than that, it looks good to merge! I unfortunately still haven't written a proper test suite for this, but from fetching your branch and running locally, this seems to fix the TTY issue as mentioned in #4.

File: src/client.rs (hunk 1 above)
Comment (leiserfg) [reply src/client.rs]: I rebased to get the `Rust 2018` update.

Comment (saghm) [reply src/client.rs]: Thanks!

**Pull request merged**